### DISCUSSION TITLE
Clumping in SP mode.

### DISCUSSION THREAD

**Discussion by rosiealice** [NGEET/fates#1044] [General]
Hi all. 

A while back we discussed how to interpret the clumping parameter when using SP mode driven with MODIS LAI.
At that point we concluded that either we should run without clumping, (the para,ref set to 1.0, which is what I have been doing) or that (to keep the parameter and RT consistent between modes) when we use SP mode, we should divide the oncoming LAI by the clumping factor (i.e. increase it), to ensure that we are not 'double counting' the clumping. 

I remember @mpaiao had insight into how MODIS account for clumping. I feel like with more people using SP mode we/I should fix this! 

@mpaiao can you remind us which way around the logic went? As in, does MODIS report the observed/ clumped LAI, or a post processed unclumped value? (I guess the former, based on the conclusion from our previous discussion) 

Thanks!

**Comment by mpaiao** [comment]
@rosiealice I checked both the [Algorithm Theoretical Basis Document](https://modis.gsfc.nasa.gov/data/atbd/atbd_mod15.pdf) for the MODIS LAI product and some publications that use it, and my understanding is that they assign biome-specific clumping indices when estimating LAI. So the value MODIS reports is the "true" LAI, not "effective" LAI. Because their estimate is for true LAI, I think we should compare FATES' LAI directly (i.e., no clumping index correction).


That said, if we had their table of clumping indices from MODIS, we could also compare the effective LAI, which could be an additional benchmark — it would help to tell whether discrepancies are due to clumping or due to the effective LAI "carrying capacity". I tried a quick search for the MODIS clumping indices, but I couldn't find the actual values.
